ftml/src/tree/align.rs

```diff
@@ -39,6 +39,15 @@ impl Alignment {
             Alignment::Justify => "justify",
         }
     }
+
+    pub fn html_class(self) -> &'static str {
+        match self {
+            Alignment::Left => "alignleft",
+            Alignment::Right => "alignright",
+            Alignment::Center => "aligncenter",
+            Alignment::Justify => "alignjustify",
```

Comment:
Just a thought, how come these are in tree instead of render/html?

Reply:
Because they're methods on AST structures. See also `HeadingLevel::html_tag()`, `AnchorTarget::html_attr()`, etc.